patches/minecraft/net/minecraft/world/biome/Biome.java.patch

```diff
@@ -16,7 +16,7 @@
 -      })).apply(p_235064_0_, Biome::new);
 +      }), ResourceLocation.field_240908_a_.fieldOf("forge:registry_name").forGetter(Biome::getRegistryName))
 +      .apply(p_235064_0_, (climate, category, depth, scale, effects, gen, spawns, name) ->
-+          net.minecraftforge.common.ForgeHooks.enhanceBiome(name, climate, category, depth, scale, effects, gen, spawns, p_235064_0_, Biome::new));
++          net.minecraftforge.common.ForgeHooks.enhanceBiome(name, new net.minecraftforge.common.world.BiomeClimateBuilder(climate.field_242460_b, climate.field_242461_c, climate.field_242462_d, climate.field_242463_e), category, depth, scale, effects, gen, spawns, p_235064_0_, Biome::new));
```

Comment:
This should be inside the hook like everything else is...

Reply:
The reason I did it this way was to avoid ATing and/or patching accessors for all the fields in `Biome.Climate` but I can change it if that's preferred.

Reply:
They are final fields, so you could just AT them. It is better practice to add getters, but this is minecraft we have to have a happy middle.

Reply:
If that's the case, I could axe the entire builder, AT all the fields and expose the constructor - would that be a preferred solution (for this and potentially `BiomeAmbiance`)? Or would it be better to expose a builder in the event rather than not.

Reply:
Im not opposed to the builder, as it lets people just edit one value without having to rebuild the object. But it is a large section of code that we'd have to keep maintaining. 
So I think public {still final} fields, and public constructor is good for now.
If modders wanna replace just one value they can copy the others from the existing one in the event.